### PR TITLE
WordPress.com Features: Don't load admin color schemes if Jetpack is not active

### PR DESCRIPTION
**Pull request by arthur791004**
Fixes https://github.com/Automattic/dotcom-forge/issues/6695

## Proposed changes:

* We haven't moved the color scheme CSS files to jetpack-mu-wpcom yet, so this PR is to disable the feature if Jetpack is not active to avoid the fatal errors.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Follow https://github.com/Automattic/jetpack/pull/37233#issuecomment-2095632582 to apply this change to your WoA site
* Deactivate Jetpack plugin
* Go to any screen on WP Admin
* Make sure the fatal errors won't be shown
*

### CODE REVIEW

File: projects/packages/jetpack-mu-wpcom/src/features/admin-color-schemes/admin-color-schemes.php [hunk 1]
```diff
@@ -167,7 +167,7 @@ function enqueue_color_scheme_for_sidebar_notice() {
 	}
 }
 
-if ( function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled() ) {
+if ( defined( 'JETPACK__PLUGIN_FILE' ) && function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled() ) {
```
Comment (jeherve) [inline projects/packages/jetpack-mu-wpcom/src/features/admin-color-schemes/admin-color-schemes.php:170]: I wonder if we should instead check for something more specific, so when the Admin color schemes feature is moved from the Masterbar module in Jetpack to the mu-wpcom, this continues to work. 
Maybe check for `Admin_Color_Schemes`?
https://github.com/Automattic/jetpack/blob/4aeebabd091a604c5135aff9053b7033fafde050/projects/plugins/jetpack/modules/masterbar.php#L31

Comment (arthur791004) [reply projects/packages/jetpack-mu-wpcom/src/features/admin-color-schemes/admin-color-schemes.php:170]: Yes, that makes sense to me. But I don't think we will rely on the `Admin_Color_Schemes` class after the feature is moved to the mu-wpcom 😅

**Review by okmttdhr — APPROVED** [review]
LGTM

we should be able to remove the condition when we move everything into jetpack-mu-wpcom

File: projects/packages/jetpack-mu-wpcom/src/features/admin-color-schemes/admin-color-schemes.php (hunk 1 above)
Comment (arthur791004) [reply projects/packages/jetpack-mu-wpcom/src/features/admin-color-schemes/admin-color-schemes.php:170]: I'm going to merge this and we can iterate the condition later or remove it as https://github.com/Automattic/jetpack/pull/37233#pullrequestreview-2042144877 mentioned 🙂

**Pull request merged**